app/furniture/marketplace/stripe_events_controller.rb

```diff
@@ -23,9 +23,7 @@ def create
         order.update!(status: :paid, placed_at: DateTime.now, payment_processor_fee_cents: balance_transaction.fee)
         order.events.create(description: "Payment Received")
 
-        if marketplace.square_order_notifications_enabled?
-          order.send_to_square_seller_dashboard
-        end
+        marketplace.square_connection&.send_order
```

Comment:
Very tiny!

Reply:
It is. Although I'd still prefer a more object-oriented, explicit exposure and/or handling of `nil` cases. 😅